src/System.Data.SqlClient/ref/System.Data.SqlClient.NetCoreApp.cs

```diff
@@ -5,6 +5,19 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+namespace Microsoft.SqlServer.Server
+{
+    [System.AttributeUsageAttribute((System.AttributeTargets)(10624), AllowMultiple=false, Inherited=false)]
+    public partial class SqlFacetAttribute : System.Attribute
```

Comment:
I defer to @AfsanehR or her colleagues for guidance on that. Since it's just a data-holding object I assumed it did not relate to any major new functionality.

Reply:
@AfsanehR any update -- I think this is the only remaining feedback. I don't really know how SqlFacetAttribute is used.

Reply:
@danmosemsft  Any specific reason that this should target netcoreapp only? This can go into the netstandard contract. Am I missing anything ? 

A note about this attribute:
SQL Server supports UDT types where a C# object can be annotated with some attributes and it can be used in Sql operations. This attribute is used to decorate the object and is interpreted by SQL Server to use in T-Sql queries. 

Currently SQL Server can deploy UDTs created in NetFx, however to decode them on Core, the UDT object needs to be referenced in Core as well. This change by @danmosemsft  will allow the UDTs to be compiled in .Net Core so that they can be fetched from SQL Server supporting SQL CLR.

@weshaggard  I didn't quite understand your concern about isolation, rather I didn't understand the term isolation in case of this attribute.

Reply:
@safern and I discussed this and believe it needed to go into netcoreapp because it is already shipped in-box in UAP (directory.build.props in parent folder has `<IsUAP>true</IsUAP>`) and therefore we did not want anyone building against this targeting net standard to be able to use this new type because at runtime it would fail on UAP. They need to indicate they're not targeting UAP, ie., targeting netcoreapp.
@weshaggard is that a correct reasoning?

Reply:
Yes that is correct.

Reply:
>@weshaggard I didn't quite understand your concern about isolation, rather I didn't understand the term isolation in case of this attribute.

My comment should read I'm not sure if it makes sense to add this in isolation. So I'm not sure if this type should go together with a set of other types or runtime feature or if it can stand on its own. In other words who will be reading this attribute and doing something accordingly?

Reply:
@saurabh500 @AfsanehR ? This is the only remaining feedback.

Reply:
@danmosemsft  Yes this is the only feedback. Thanks for clarifying the netstandard issue.
@weshaggard  The attribute is read by SQL Server.

Reply:
@saurabh500 can you clarify how it is ready by sql server? I would expect something in System.Data.SqliClient to read and interpret this attribute when communicating with the server.

Reply:
@weshaggard The attribute is used on C# Types that are deployed on SQL Server. These annotated C# types (User Defined CLR types) on SQL Server can be used by SQL Server to then populate the data on the return types, based on the attribute specification, before returning data as C# objects.

Reply:
Do you sign off? 😄